pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -72,6 +72,9 @@ python3.pkgs.buildPythonApplication rec {
         "docs/yaml/objects/dep.yaml"
       ];
     })
+
+    # Fix detection of CMAKE_SIZEOF_VOID_P
+    ./CMAKE_SIZEOF_VOID_P.patch
```

Comment:
Do you think you could `fetchpatch` this instead of vendoring the patch file into the tree?

(should be able to use a URL like <https://github.com/mesonbuild/meson/commit/e12b5c1474701cac81cea75f1d8215ff3a11d9bf.patch>)

Also could you mention the upstream meson PR url in this comment?

Thank you for figuring this out!

Reply:
Thanks for the feedback, I have changed it to using fetchpatch instead

Reply:
The PR is now merged upstream, so you do not need to use a `pull/11761/commits/` style link for fetchpatch anymore. The new commit sha1 is [7c78c2b5a0314078bdabb998ead56925dc8b0fc0](https://github.com/mesonbuild/meson/commit/7c78c2b5a0314078bdabb998ead56925dc8b0fc0).